Applications/ConsumerApi/src/Configuration/ConsumerApiConfiguration.cs

```diff
@@ -6,12 +6,12 @@ namespace Backbone.ConsumerApi.Configuration;
 public class ConsumerApiConfiguration
```

Comment:
In this file there are a lot of required properties that are not marked as required. Is there a specific reason for that?